src/includes/utils/localTime.hpp

```diff
@@ -0,0 +1,19 @@
+//
+// Created by pmerku on 3/20/2021.
+//
+
+#ifndef LOCALTIME_HPP
+#define LOCALTIME_HPP
+
+#include <string>
+
+namespace utils {
+
+	inline struct tm *relativeLocalTime(const time_t *t, struct tm *tm) {
+		(void)t;
+		return tm;
+	}
```

Comment:
is this unfinished? what will it be doing?

Reply:
Yes unfinished, this function will populate the time struct for strftime()